geometry_msgs/Point[] position
geometry_msgs/Point[] velocity
geometry_msgs/Point[] acceleration
float64[] yaw
float64[] yaw_rate
uint32[] wait_freq